index.js

```diff
@@ -23,7 +20,6 @@ module.exports = {
   extends: [
     "plugin:react/recommended",
     "plugin:@typescript-eslint/eslint-recommended",
-    "prettier/@typescript-eslint",
```

Comment:
Did this get rolled into prettier below?

Reply:
yes https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

Reply:
Oh that's awesome!